### ISSUE TITLE
License headers should not show up on module-level docs

### ISSUE THREAD

**Issue by roj1512** [denoland/deno_doc#550]

**Comment by iuioiua** [comment]
Please elaborate and provide an example of where you're seeing this.

**Comment by roj1512** [comment]
https://jsr.io/@mtkruto/mtkruto/doc/~/MtkrutoError

**Comment by roj1512** [comment]
No symbol has this doc.

![image](https://github.com/denoland/deno_doc/assets/49933115/e69a9b67-eba6-4be2-a4e4-73e50c6496e3)

**Comment by iuioiua** [comment]
This behaviour is expected. `deno_doc` sees this text as a plain JSDoc description. Currently, the way to do this is to prefix each line of the comment with `//` instead of wrapping it in `/** ... */`. Either way, supporting the [`@license` JSDoc tag](https://jsdoc.app/tags-license) would be nice if someone wants to display their license.

**Comment by roj1512** [comment]
Did I understand this wrongly or this module-level doc used to be explicitly enabled with @module?

**Comment by iuioiua** [comment]
Yes, I do believe the `@module` tag is required to denote module-level docs. But do not confuse what you shared (https://jsr.io/@mtkruto/mtkruto/doc/~/MtkrutoError) as a module doc. It's a symbol doc, for the `MtkrutoError` class.

**Comment by roj1512** [comment]
Oh, I just understood.
